platform/stm32f4xx/export/platform/spi_bus.hpp

```diff
@@ -756,7 +793,7 @@ constexpr uint32_t spi_i2s_bus< spi_i2s_config >::pick_i2s_data_format(ecl::i2s:
 }
 
 template< class spi_i2s_config >
-void spi_i2s_bus< spi_i2s_config >::init_dma()
+void spi_i2s_bus< spi_i2s_config >::init_irq()
 {
     dma::init_rcc< spi_i2s_config::m_dma_rx_stream >();
     dma::init_rcc< spi_i2s_config::m_dma_tx_stream >();
```

Comment:
Let's move these two lines (`init_rcc()`) somewhere else, i.e. to the new `init_clk()` routine and add there [clock initialization of the bus itself](https://github.com/forGGe/theCore/pull/101/files#diff-2eafd263ee479c3af12971e7a5766601R339)